project/plugins.sbt

```diff
@@ -1,8 +1,14 @@
 scalacOptions ++= Seq("-deprecation", "-feature")
 
-addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.3.0")
-addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.7.0")
-addSbtPlugin("com.typesafe" % "sbt-sdlc" % "0.1")
-addSbtPlugin("de.johoop" % "sbt-testng-plugin" % "3.1.1")
-addSbtPlugin("com.novocode" % "sbt-ornate" % "0.5")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.2")
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.7.0")
```

Comment:
a newer MiMa is available -- I guess that could be a separate PR

Reply:
I've set the latest version and for sbt-pgp